src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -1054,14 +1054,12 @@ private String swapStore(String storeName, String directory) throws VoldemortExc
                                                                                                             .setStatus("started");
         try {
             if(!metadataStore.getReadOnlyFetchEnabledUnlocked()) {
-                throw new ReadOnlyFetchDisabledException("Read-only fetcher is disabled in "
-                                             + metadataStore.getServerStateUnlocked()
-                                             + " state on node " + metadataStore.getNodeId());
+                throw new ReadOnlyFetchDisabledException("Pushes to this node have been administratively disabled."
```

Comment:
Sure. I think it's fine to leave the second line of your new error message. But I definitely think we should not make the error message over-specific by claiming that this is due to an administrative operation when it is not necessarily the case.